simple-backend/stave_backend/lib/stave_pack_parser.py

```diff
@@ -0,0 +1,383 @@
+"""
+Provide interfaces to parse DataPack and ontology for interaction and
+conversion of different data formats between frontend side and backend side.
+"""
+
+import json
+from typing import Dict, Optional
+from packaging.version import Version
+import tempfile
+
+from forte.data import DataPack, DataStore
+from forte.data.ontology import Annotation, Link
+from forte.common import constants
+from forte.version import PACK_ID_COMPATIBLE_VERSION
+
+
+class StavePackParser:
+    r"""
+    StavePackParser provides interfaces to parse DataPack and ontology
+    specifications for backend APIs to interact with frontend data formats.
+    Example usage:
+
+            StavePackParser(raw_pack, raw_ontology).transform_pack()
+
+    """
+    
+    ANNOTATION_LIST = "annotations"
+    LINK_LIST = "links"
```

Comment:
This is because `Stave` users can only add/edit/delete these two types of top ontologies. Looks like we haven't design any mechanism for users to manage `Group` or other types of ontologies through `Stave` UI.